packages/esm-billing-app/src/benefits-package/forms/benefits-eligibility-request-form.workspace.tsx

```diff
@@ -72,187 +72,202 @@ const BenefitsEligibilyRequestForm: React.FC<BenefitsEligibilyRequestFormProps>
+                      subtitle={t('errorFetchingPackages', 'Error fetching packeges')}
+                      lowContrast
+                    />
+                  ) : (
+                    <Dropdown
+                      ref={field.ref}
+                      invalid={form.formState.errors[field.name]?.message}
+                      invalidText={form.formState.errors[field.name]?.message}
+                      id="package"
+                      titleText={t('package', 'Package')}
+                      onChange={(e) => {
+                        field.onChange(e.selectedItem);
+                      }}
+                      initialSelectedItem={field.value}
+                      label="Choose package"
+                      items={packages.map((r) => r.uuid)}
+                      itemToString={(item) => packages.find((r) => r.uuid === item)?.packageName ?? ''}
+                    />
+                  )}
+                </>
+              )}
+            />
+          </Column>
+          {selectedPackageObservable && (
+            <Column>
+              <PackageIntervensions
+                category={packages.find((package_) => package_.uuid === selectedPackageObservable)?.packageCode ?? ''}
               />
-            )}
-          />
-        </Column>
-        <Column>
-          <Controller
-            control={form.control}
-            name="interventions"
-            render={({ field }) => (
-              <MultiSelect
-                ref={field.ref}
-                invalid={form.formState.errors[field.name]?.message}
-                invalidText={form.formState.errors[field.name]?.message}
-                id="interventions"
-                titleText={t('interventions', 'Interventions')}
-                onChange={(e) => {
-                  field.onChange(e.selectedItems);
-                }}
-                initialSelectedItems={field.value}
-                label="Choose option"
-                items={interventions.map((r) => r.interventionCode)}
-                itemToString={(item) => interventions.find((r) => r.interventionCode === item)?.interventionName ?? ''}
-              />
-            )}
-          />
-        </Column>
-        <Column>
-          <Controller
-            control={form.control}
-            name="isRefered"
-            render={({ field }) => (
-              <Checkbox
-                id="isRefred"
-                labelText="Is Referred"
-                checked={field.value}
-                onChange={(_, { checked }) => field.onChange(checked)}
-              />
-            )}
-          />
-        </Column>
-      </Stack>
+            </Column>
+          )}
+          <Column>
+            <Controller
+              control={form.control}
+              name="diagnosisUuids"
+              render={({ field }) => (
+                <MultiSelect
+                  ref={field.ref}
+                  invalid={form.formState.errors[field.name]?.message}
```

Comment:
nit
```suggestion
                  invalid={errors[field.name]?.message}
```
we could extract the `errors` property from `formState` to make this less verbose.